config/routes.rb

```diff
@@ -1,4 +1,6 @@
 Rails.application.routes.draw do
+  default_url_options :host => "example.com"
```

Comment:
Hashrockets is so 2017... :) Please upgrade this piece of code

Reply:
Ofc we will!